addon/components/nrg-datetime/calendar/component.js

```diff
@@ -67,8 +67,8 @@ export default Component.extend(EKMixin, EKFirstResponderOnFocusMixin, {
         year: this.selectedYearIndex,
       });
       if (this.isSelectingYears) {
-        const firstYear = this.years[0][0].year;
-        const lastYear = this.years[4][1].year;
+        const firstYear = this.years[0][0].display;
+        const lastYear = this.years[4][1].display;
```

Comment:
`year` is an object now, so I used the 'display' property instead because I feel it's more readable than `years[0][0].year.year`.